packages/components/Organisms/SiteFooter/Acknowledgement.vue

```diff
@@ -1,12 +1,18 @@
 <template>
-  <div class="rpl-acknowledgement" v-if="text" >{{ text }}</div>
+  <div class="rpl-acknowledgement" v-if="text">
+    <img class="rpl-acknowledgement__flag" src="./assets/images/aboriginal-flag.png" :alt="aboriginalFlagAltText" width="53px" height="36px" />
+    <img class="rpl-acknowledgement__flag" src="./assets/images/torres-strait-islander-flag.png" :alt="torresStraitIslanderFlagAltText" width="53px" height="36px" />
```

Comment:
Should we use css for the img width and height instead of image attributes?

Reply:
Yep, I can update this.